components/app/project-detail/ProjectSingleHeader.vue

```diff
@@ -1,43 +1,42 @@
 <template>
   <LandingSectionContainer>
-    <div class="flex justify-between content-center">
-      <div class="flex">
-        <div class="mr-4 w-12 h-12">
-          <app-logo
-            class="inline-block rounded-full ring-2 ring-white"
-            :src="project.logo"
-            :alt="project.title"
-          />
-        </div>
+    <div
+      class="flex flex-wrap items-center justify-between gap-4 xl:flex-no-wrap"
```

Comment:
suggestion(blocking): dont use flex gap since its browser support is still not so good